### ISSUE TITLE
Feature request: Adjust windows following the Golden Ratio

### ISSUE THREAD

**Issue by gabridome** [eczarny/spectacle#311]
http://en.wikipedia.org/wiki/Golden_ratio.

I personally think it would be nice to work in an environment in which your open task are balanced in a golden ratio way. 

An alternative good solution might be to be able to customize the ratio in which the tiled windows must relate to each other.

**Comment by azdawg** [comment]
Yeah, I think that's a cool idea, I would also like to add 'Make Longer" & 'Make Shorter'.
Btw: this is a great utility!

**Comment by eczarny** [comment]
Interesting idea! I would love some more examples to get a better idea of the feature.

**Comment by gabridome** [comment]
Until now Spectacle change the window according to a command and in relation to the entire screen. This feature is challenging because the window should be put in relation to the screen and an other window.
Maybe the other window to be put in relation with could be the next in the tasks list. An other solution could be to let the user choose the two windows to put in relation but this would raise usability problems.
An other choice is whether the window you are sizing and positioning is the bigger or the smaller of the pair but I think it should be the bigger.

So to sum up a solution might be: the window you are focusing on should be sized As "a" and positioned to the left side of the screen and the next task in the tasks list should be positioned on the right with size "b" to fill the screen completely. "a"  and "b" are the size calculated as in http://en.wikipedia.org/wiki/Golden_ratio
